iot-hub/Samples/device/X509DeviceCertWithChainSample/X509DeviceCertWithChainSample.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.Azure.Devices.Client;
```

Comment:
Not sure if we need a copyrights header here. Maybe check other samples

Reply:
Good point. Will check.